src/RestrictedPython/compile.py

```diff
@@ -19,9 +24,13 @@ def _compile_restricted_mode(
         flags=0,
         dont_inherit=False,
         policy=RestrictingNodeTransformer):
+
+    if not IS_CPYTHON:
+        warnings.warn(NOT_CPYTHON_WARNING, RuntimeWarning)
+
     byte_code = None
-    errors = []
-    warnings = []
+    collected_errors = []
+    collected_warnings = []
```

Comment:
What does this refactoring have to do with the "not on PyPy" purpose of the PR?

Reply:
The `warnings` variable shadows the `warnings` module which is now needed some lines after the variable definition. I did not want to change only the name of one variable.